modules/ROOT/pages/cloudhub-architecture.adoc

```diff
@@ -104,8 +104,7 @@ Workers with 1 or more vCores:
 * Provide performance consistency
 * Are available only for Mule version 3.6.2 or later, or API gateway version 2.0.2 or later
 
-Each worker has a minimum of 8 GB of storage, which is used for both system and application storage.
-If your application requires more storage (for example, verbose logging), use a worker with two or more vCores, enabling the application to access additional storage in `/tmp`.
+Each worker has a minimum of 8 GB of storage allocated which is used for both system and application storage with about 3.0 GB is used for OS and Mule components. If your application requires more storage (for example, verbose logging), use a worker with two or more vCores, enabling the application to access additional storage
```

Comment:
Each worker has a minimum of 8 GB of storage allocated, which is used for both system and application storage, with about 3.0 GB used for OS and Mule components. If your application requires more storage (for example, verbose logging), use a worker with two or more vCores, enabling the application to access additional storage.